charts/sck-otel/values.yaml

```diff
@@ -27,19 +27,20 @@ splunkPlatform:
   # HTTP timeout when sending data. Defaults to 10s.
   timeout: 10s
   # Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS. Defaults to false.
-  insecure_skip_verify: false
-  # The PEM-format CA certificate for this client.
-  # NOTE: The content of the certificate itself should be used here, not the file path.
-  #       The certificate will be stored as a secret in kubernetes.
-  clientCert:
-  # The private key for this client.
-  # NOTE: The content of the key itself should be used here, not the file path.
-  #       The key will be stored as a secret in kubernetes.
-  clientKey:
-  # The PEM-format CA certificate file.
-  # NOTE: The content of the file itself should be used here, not the file path.
-  #       The file will be stored as a secret in kubernetes.
-  caFile:
+  tls:
+    insecure_skip_verify: false
+    # The PEM-format CA certificate for this client.
+    # NOTE: The content of the certificate itself should be used here, not the file path.
+    #       The certificate will be stored as a secret in kubernetes.
+    clientCert:
+    # The private key for this client.
+    # NOTE: The content of the key itself should be used here, not the file path.
+    #       The key will be stored as a secret in kubernetes.
+    clientKey:
+    # The PEM-format CA certificate file.
+    # NOTE: The content of the file itself should be used here, not the file path.
+    #       The file will be stored as a secret in kubernetes.
+    caFile:
```

Comment:
let's not wrap it under `tls` here in `values.yaml` file. it just doesn't need to be. 
in this way, we don't have to change all references to `.Values.splunkPlatform.clientKey` and others don't need to be updated.

Reply:
@rockb1017 is this error related to us not using the latest contrib image in our distro which is missing `tls` config update?